live-examples/js-examples/intl/intl-listformat-prototype-resolvedoptions.js

```diff
@@ -0,0 +1,12 @@
+const deListFormatter1 = new Intl.ListFormat('de-DE', { style: 'short', type: 'disjunction' });
```

Comment:
```suggestion
const deListFormatter1 = new Intl.ListFormat('de-DE', { type: 'disjunction' });
```

maybe, so we can show the defaults too? (of course you'd need to change the expected output as well)

Reply:
Alright added :) plus changed the comment to long for the style console log.